### PR TITLE
General Usability Bugfixes

### PR DESCRIPTION
**Pull request by jujaga**
# Description

- Enable gzip compression for webpack dev server
- Ensure share link warning appears when no forms are published
- Make v-alert styles more consistent across the app (transitions, primary/info color)
- Add help text alert while editing submission (staff side)
- Update user search drop down value text when no data available
- Change submission list icon to list_alt
- Fix copy to clipboard button and snackbar consistency
- Add missing tooltips to view action button
- Change export/download icon to match other export buttons
- Update assignment label text to be more precise
- Disable Status and Notes panel when editing submission (staff side)

[SHOWCASE-2106](https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-2106)

## Types of changes

Bug fix (non-breaking change which fixes an issue)

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have checked that unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by bcgov-nr-csst** [comment]
PR Deployed at: https://chefs-dev.apps.silver.devops.gov.bc.ca/pr-189

File: app/frontend/src/components/forms/manage/AddTeamMember.vue [hunk 1]
```diff
@@ -13,6 +13,11 @@
         return-object
         :search-input.sync="searchUsers"
       >
+        <!-- no data -->
+        <template #no-data>
+          <span>Can't find someone? They may not have joined the site.<br/>
+            Kindly send them a link to the site and ask them to log in.</span>
+        </template>
```
Comment (loneil) [inline app/frontend/src/components/forms/manage/AddTeamMember.vue:20]: Little janky without padding, could class the span and add style to this component for padding.
But does just using the v-autocomplete `no-data-text` attribute look better than the slot?

image

Comment (jujaga) [reply app/frontend/src/components/forms/manage/AddTeamMember.vue]: Added `px-2` padding to it.

**Review by loneil — APPROVED** [review]

File: app/frontend/src/components/forms/manage/ManageFormActions.vue [hunk 2]
```diff
@@ -93,13 +93,17 @@ export default {
     canViewSubmissions() {
       const perms = [
         FormPermissions.SUBMISSION_READ,
-        FormPermissions.SUBMISSION_UPDATE
+        FormPermissions.SUBMISSION_UPDATE,
```
Comment (MichaelChung123) [inline app/frontend/src/components/forms/manage/ManageFormActions.vue:96]: is this comma required?

**Review by MichaelChung123 — APPROVED** [review]
LGTM

File: app/frontend/src/components/forms/manage/ManageFormActions.vue (hunk 2 above)
Comment (jujaga) [reply app/frontend/src/components/forms/manage/ManageFormActions.vue:96]: No it's not, and personally prefer it to not be there, but our VSCode autoformatter seems to insist these trailing end commas be there.

**Pull request merged**